### ISSUE TITLE
fix predict function in s2s 

### ISSUE THREAD

**Issue by outformatics** [facebookresearch/ParlAI#1120]
- In predict function, rename valid_cands => cand_indices

**Comment by outformatics** [comment]
Fix the issue so the following example can get executed.

> Train a seq2seq model on the "10k training examples" bAbI task 1 with batch size of 32 examples until accuracy reaches 95% on validation (requires pytorch):
`python examples/train_model.py -t babi:task10k:1 -m seq2seq -mf /tmp/model_s2s -bs 32 -vtim 30 -vcut 0.95`
